x-pack/plugins/search_playground/server/utils/get_chat_params.ts

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
This seems like it should be a `lib` file and not a `utils` file IMO

Reply:
agreed. will rename that in a focused PR.